libvast/test/arrow_extension_types.cpp

```diff
@@ -66,3 +70,103 @@ TEST(subnet type serde roundtrip) {
 TEST(pattern type serde roundtrip) {
   serde_roundtrip<vast::pattern_extension_type>();
 }
+
+template <class... T>
+auto is_type() {
+  return []<class... U>(const U&...) {
+    return (std::is_same_v<T, U> && ...);
+  };
+}
```

Comment:
This needs to be wrapped in an anonymous namespace to avoid ODR problems. Please also check if there are any other free functions that are local to this translation unit that are not yet wrapped in one.

Reply:
done for the entire unit